src/wallet/api/walletApi.ts

```diff
@@ -219,6 +227,47 @@ const walletApi = {
       throw new WalletRequestError('Error requesting auth token.');
     }
   },
+
+  async pushRegister(wallet: HathorWalletServiceWallet, payload: PushRegisterRequestData): Promise<PushRegisterResponseData> {
+    const axios = await axiosInstance(wallet, true);
+
+    const response = await axios.post<PushRegisterResponseData>('push/register', payload);
+    if (response.status === 200 && response.data.success) {
+      return response.data;
+    } else {
+      throw new WalletRequestError('Error registering device for push notification.', { cause: response.data });
+    }
+  },
+
+  async pushUpdate(wallet: HathorWalletServiceWallet, payload: PushUpdateRequestData): Promise<PushUpdateResponseData> {
+    const axios = await axiosInstance(wallet, true);
+    const response = await axios.put<PushUpdateResponseData>('push/update', payload);
+    if (response.status === 200 && response.data.success) {
+      return response.data;
+    } else {
+      throw new WalletRequestError('Error updating push notification settings for device.', { cause: response.data });
+    }
+  },
+
+  async pushUnregister(wallet: HathorWalletServiceWallet, payload: PushUnregisterRequestData): Promise<PushUnregisterResponseData> {
+    const axios = await axiosInstance(wallet, true);
+    const response = await axios.post('push/unregister', payload);
```

Comment:
Should we have here `<PushUnregisterResponseData>`?

Reply:
No. I've refactor it to send a `DELETE` and the `deviceId` through path param.